backend/app.py

```diff
@@ -14,41 +14,35 @@
 CACHE_CONFIG = {"CACHE_TYPE": "filesystem", "CACHE_DIR": "/tmp"}
 cache = Cache(app, config=CACHE_CONFIG)
 
-AIRTABLE_API = "https://api.airtable.com/v0/"
-AIRTABLE_BASE = "appISQqfvRPMg6CH3/Suivi"
-API_KEY = os.getenv("AIRTABLE_API_KEY")
-HEADERS = {"Authorization": f"Bearer {API_KEY}"}
+NOTION_API = "https://api.notion.com/v1/databases/"
+NOTION_BASE = os.getenv("NOTION_BASE")
+API_KEY = os.getenv("NOTION_API_KEY")
+HEADERS = {
+    "Authorization": f"Bearer {API_KEY}",
+    "Content-Type": "application/json",
+    "Notion-Version": "2022-06-28"
+}
 CACHE_TIMEOUT = 1 if app.config['DEBUG'] else os.getenv('CACHE_TIMEOUT', 600)
 
 
 @app.route("/api/inventaire")
 @cache.cached(timeout=CACHE_TIMEOUT)
 def inventaire():
-    url = f"{AIRTABLE_API}{AIRTABLE_BASE}"
-    r = requests.get(url, headers=HEADERS)
-    return jsonify(r.json())
-
-
-@app.route("/api/demande", methods=["POST"])
-def demande():
-    # mapping {<form_name>: <airtables_name>}
-    fields = {
-        "description": "Données recherchées",
-        "title": "Titre du jeu de données",
-        "organisation": "Organisation",
-        "geo": "Echelle géographique souhaitée",
-        "update": "Fréquence de mise à jour souhaitée",
-        "reuse": "Réutilisation envisagée",
-        "contact": "Contact",
+    url = f"{NOTION_API}{NOTION_BASE}/query"
+    payload = {
+        "page_size": 100,
+        "filter": {
+            "property": "Public",
+            "checkbox": {
+                "equals": True
+            }
+        },
     }
-    url = f"{AIRTABLES_URL}/Demandes%20d'ouverture"
-    fields_dict = {fields[fkey]: request.json.get(fkey) for fkey in fields.keys()}
-    fields_dict["Date"] = date.today().isoformat()
-    r = requests.post(url, json={
-        "fields": fields_dict
-    }, headers=HEADERS)
-    r.raise_for_status()
-    return "", 201
+    start_cursor = request.args.get("start_cursor")
+    if start_cursor:
+        payload["start_cursor"] = start_cursor
+    r = requests.post(url, json=payload, headers=HEADERS)
+    return jsonify(r.json())
```

Comment:
We should probably deal with errors between Notion API - backend - frontend and give minimal feedback to the user.
You could use `r.raise_for_status()` for example before returning here